### ISSUE TITLE
Is EventStore support production ready?

### ISSUE THREAD

**Issue by rasmus** [eventflow/EventFlow#469]
Can I remove the "only for test (for now)" for EventStore in the README.md?

#### tl;dr

The current implementation of EventStore was something I made as a PoC and thus I haven't personally verified the implementation in a production environment.

This is a call-out to anyone either currently using the implementation or someone experienced in working with EventStore to verify that the implementation is "production ready".

**Comment by volak** [comment]
I just use EventStore - the code looks alright except you don't need to use transactions here: https://github.com/eventflow/EventFlow/blob/4420e2eb3a026b9badc900006d807f5d41416c56/Source/EventFlow.EventStores.EventStore/EventStoreEventPersistence.cs#L135

They are only used to write multiple times in batches and can be slower than just doing `AppendToStreamAsync` like normal.

In my library I used to only use transactions if the events to write exceeded 100 events or so.  But then I saw some post by greg or someone that said they were thinking about removing transactions all together.  
I might be wrong though you will have better response in the cqrs/ddd slack group I think
